src/inlay_hints.ts

```diff
@@ -124,14 +124,20 @@ export class HintsUpdater implements Disposable {
           continue;
       }
     }
+    console.info(JSON.stringify(decorations));
+    console.info(doc);
```

Comment:
Remove the logs please.

Reply:
Oops, my bad. Removed! Please check the new version :)